style/style-global.js

```diff
@@ -35,11 +36,22 @@ export default css.global`
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     word-break: break-word;
-    font-size: 20px;
```

Comment:
I feel like we are going back and forth with changing global styles. It would be great if we could agree on a set of global styles and then try not to change them unless it really makes sense. Do you like to chat about your changes here?

Reply:
Oh! I didn't realize you also change this part. I remember you told me you find the font too big but i don't remember changes so i did.
Sure, let's talk about it as soon as we are both in slack :)